tests/databaseDiff.js

```diff
@@ -0,0 +1,253 @@
+                description2 = description;
+                callback(null);
+            });
+        },
+        (callback) => {
+            // Determine if both databases have the same tables
+            let missingFrom1 = _.difference(_.keys(description2.tables), _.keys(description1.tables));
+            let missingFrom2 = _.difference(_.keys(description1.tables), _.keys(description2.tables));
+
+            if (missingFrom1.length > 0) {
+                let db1NameBold = formatText(db1Name, colors.bold);
+                result += formatText(`Tables missing from ${db1NameBold} (${db1.type})\n`, colors.underline);
+                result += missingFrom1.map(table => `    ${table}`).join('\n') + '\n\n';
+            }
+
+            if (missingFrom2.length > 0) {
+                let db2NameBold = formatText(db2Name, colors.bold);
+                result += formatText(`Tables missing from ${db2NameBold} (${db2.type})\n`, colors.underline);
+                result += missingFrom2.map(table => `    ${table}`).join('\n') + '\n\n';
+            }
+
+            callback(null);
+        },
+        (callback) => {
+            // Determine if both databases have the same enums
+            let missingFrom1 = _.difference(_.keys(description2.enums), _.keys(description1.enums));
+            let missingFrom2 = _.difference(_.keys(description1.enums), _.keys(description2.enums));
+
+            if (missingFrom1.length > 0) {
+                let db1NameBold = formatText(db1Name, colors.bold);
+                result += formatText(`Enums missing from ${db1NameBold} (${db1.type})\n`, colors.underline);
+                result += missingFrom1.map(enumName => `    ${enumName}`).join('\n') + '\n\n';
+            }
+
+            if (missingFrom2.length > 0) {
+                let db2NameBold = formatText(db2Name, colors.bold);
+                result += formatText(`Enums missing from ${db2NameBold} (${db2.type})\n`, colors.underline);
+                result += missingFrom2.map(enumName => `    ${enumName}`).join('\n') + '\n\n';
+            }
+
+            callback(null);
+        },
+        (callback) => {
+            // Determine if the columns of any table differ
+            let intersection = _.intersection(_.keys(description1.tables), _.keys(description2.tables));
+            _.forEach(intersection, (table) => {
+                // We normalize each blob to end with a newline to make diffs print cleaner
+                const diff = jsdiff.diffLines(description1.tables[table].trim() + '\n', description2.tables[table].trim() + '\n');
+                if (diff.length == 1) return;
+
+                const boldTable = formatText(table, colors.bold);
+                result += formatText(`Differences in ${boldTable} table\n`, colors.underline);
+                _.forEach(diff, (part) => {
+                    result += formatText(part.value, part.added ? colors.green : part.removed ? colors.red : null);
```

Comment:
On this and similar red/green lines, could we set the first character to -/+ to mimic the unified-diff format?